feat: List all annotations

philipjackson opened the pull request:
## Description

ViewAnnotationsDialog now lists all users who have made an annotation, be they collaborator, member or owner. Also fixes the title of the dialog to say "Compare Annotations" when the Compare Annotations button is clicked.

Requires https://github.com/gliff-ai/dominate/pull/753 to work

## Checklist:

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request). If you're unsure about any of these, don't hesitate to leave a comment on this pull request!

- [ ] I have read the gliff.ai Contribution Guide.
- [ ] I have requested to **pull a branch** and not from main.
- [ ] I have checked all commit message styles match the requested structure.
- [ ] My code follows the style guidelines of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have performed a self-review of my own code.
- [ ] I have assigned **3 or less** reviewers.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My changes generate no new warnings.
- [ ] I have made corresponding changes to the documentation.
- [ ] New database changes have been committed.
- [ ] If appropriate, I have bumped any version numbers.

SilviaZeta reviewed (changes requested):
I've noticed that the `usersWithAnnotations` fields is also in the sort and search dropdowns:
<img width="278" alt="Screenshot 2022-07-18 at 13 15 18" src="https://user-images.githubusercontent.com/13273585/179509098-74ad7035-a528-44f6-8a5e-50d65034d468.png">

Is this what we want?

ChasNelson1990 reviewed (changes requested):
Erm, I can't click on images anymore in the latest Azure Static App? The app crashes with an unread property error.

philipjackson commented:
> Is this what we want?
> Erm, I can't click on images anymore in the latest Azure Static App? The app crashes with an unread property error.

Fixed and fixed.
